src/client/datascience/interactive-window/interactiveWindow.ts

```diff
@@ -533,7 +534,7 @@ export class InteractiveWindow extends InteractiveBase implements IInteractiveWi
         }
 
         // Pull out the metadata from our active notebook
-        const metadata: nbformat.INotebookMetadata = { orig_nbformat: 3 };
```

Comment:
I thought we had done this on purpose. That the 4.2 would be too new. Swear there was a bug where somebody complained about not supporting 3.0.

Reply:
`orig_format` is not the same as nbformat.
Orig_format is used by Jupyter to keep track of what it used to be.
E.g. if someone opens an existig notebook that was of format 3, and it gets upgraded to 4, then this will contain `orig_format` of 3.

All notebooks we create via `create blank notebook` are of format 4.2 today, we don't support anything other than that (that's existig code where we set nbformat to `4`).
This change makes it more clear.